### PR TITLE
[forum registration] Add some more clarifying text

### PR DESCRIPTION
**Pull request by kib**
Hopefully it cuts down on the amount of registration issues.
@da-anda what do you think ?

### CODE REVIEW

**Comment by da-anda** [comment]
the only issue I have with this is that these changes will be lost on updates. So it would probably be better to override the language strings in our override file: xbmc/languages/english/xbmc.lang.php

**Comment by kib** [comment]
good point, I'll redo it.

**Comment by kib** [comment]
@da-anda How is this ?

File: xbmc/languages/english/xbmc.lang.php [hunk 1]
```diff
@@ -49,4 +49,15 @@
...
+$l['agreement_2'] = "By registering on this discussion system you agree that you will not post any material which is knowingly false, inaccurate, abusive, hateful, harassing, sexually orientated, threatening or invasive of a person's privacy, or any other material which may violate any applicable laws.";
+$l['agreement_3'] = "Failure to comply with these rules may result in the termination of your account, account suspension, or permanent ban of access to these forums. Your IP Address is recorded with each post you make on this discussion system and is retrievable by the forum staff if need-be. You agree that we have the ability and right to remove, edit, or lock any account or message at any time should it be seen fit. You also agree that any information you enter on this discussion system is stored in a database, and that "cookies" are stored on your computer to save your login information. Any information you provide on these forums will not be disclosed to any third party without your complete consent, although the staff cannot be held liable for any hacking attempt in which your data is compromised.";
```
Comment (da-anda) [inline xbmc/languages/english/xbmc.lang.php]: the quotes around "cookies" need to be escaped

**Comment by da-anda** [comment]
looks good besides of the quotes issue. Have you checked if overriding these labels actually works?

**Comment by kib** [comment]
fixed the quotes.
The myBB forums tell several people to modify these labels to change the text. I figured we could override them like any other label. Our file is loaded last I presume ?

**Pull request merged**